google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -3046,13 +3048,16 @@ public void testEnableAndDisableBucketPolicyOnlyOnExistingBucket() throws Except
       Bucket remoteBucket =
           storage.get(bpoBucket, Storage.BucketGetOption.fields(BucketField.IAMCONFIGURATION));
 
-      assertTrue(remoteBucket.getIamConfiguration().isBucketPolicyOnlyEnabled());
-      assertNotNull(remoteBucket.getIamConfiguration().getBucketPolicyOnlyLockedTime());
+      assertTrue(remoteBucket.getIamConfiguration().isUniformBucketLevelAccessEnabled());
+      assertNotNull(remoteBucket.getIamConfiguration().isUniformBucketLevelAccessEnabled());
```

Comment:
isUniformBucketLevelAccessEnabled is used twice.
please replace with uniformBucketLevelAccessLockedTime

Reply:
other looks good.

Reply:
> isUniformBucketLevelAccessEnabled is used twice.
> please replace with uniformBucketLevelAccessLockedTime

thanks for catching this

Reply:
@dmitry-fa PTAL